cmd/config-common.go

```
@@ -79,3 +119,39 @@ func checkConfig(ctx context.Context, objAPI ObjectLayer, configFile string) err
 	}
 	return nil
 }
+
+func saveXattrConfig(ctx context.Context, objAPI ObjectLayer, configFile string, data []byte) error {
+	hashReader, err := hash.NewReader(bytes.NewReader(data), int64(len(data)), "", getSHA256Hash(data), int64(len(data)), globalCLIContext.StrictS3Compat)
+	if err != nil {
+		return err
+	}
+	var buffer bytes.Buffer
+	_, err = io.Copy(&buffer, hashReader)
+	if err != nil {
+		dlogPrintf("Copy error : %+v\n", err)
```

does this need to return an error?